lib/khmer.hh

```diff
@@ -54,7 +54,7 @@ private:\
 #   define CIRCUM_MAX_VOL 200	// @CTB remove
 
 #   define SAVED_SIGNATURE "OXLI"
-#   define SAVED_FORMAT_VERSION 4
+#   define SAVED_FORMAT_VERSION 5
```

Comment:
That's only one step better but acceptable for this next release as we
broke everything

On Tue, Jul 28, 2015 at 2:41 PM C. Titus Brown <notifications@github.com>
wrote:

> In lib/khmer.hh
> <https://github.com/dib-lab/khmer/pull/1194#discussion_r35704979>:
>
> > @@ -54,7 +54,7 @@ private:\
> >  #   define CIRCUM_MAX_VOL 200	// @CTB remove
> >
> >  #   define SAVED_SIGNATURE "OXLI"
> > -#   define SAVED_FORMAT_VERSION 4
> > +#   define SAVED_FORMAT_VERSION 5
>
> On Tue, Jul 28, 2015 at 02:37:13PM -0700, Michael R. Crusoe wrote: > @@
> -54,7 +54,7 @@ private:\ > # define CIRCUM_MAX_VOL 200 // @CTB
> <https://github.com/CTB> remove > > # define SAVED_SIGNATURE "OXLI" > -#
> define SAVED_FORMAT_VERSION 4 > +# define SAVED_FORMAT_VERSION 5 Gah, do we
> need per-file-type versioning?
> Yeah, I kinda cringed. Maybe we could do *per release* file versioning, so
> this would only be incremented after a release?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dib-lab/khmer/pull/1194/files#r35704979>.
>
-- 
Michael R. Crusoe: Programmer & Bioinformatician crusoe@ucdavis.edu
The lab for Data Intensive Biology; University of California, Davis
https://impactstory.org/MichaelRCrusoe http://twitter.com/biocrusoe